### ISSUE TITLE
Handle authentication from MC by redirect to Cognito

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#302]

**Comment by mike-gangl** [comment]
if we use mgmt console to _deploy_ the httpd/proxy, then we'll need to do the auth in the MC itself.

**Comment by mike-gangl** [comment]
i don't think the management console should be open to anyone with the url. so i'd push back on the no auth part here. once in, then i agree we don't have to lock it down just yet.

**Comment by galenatjpl** [comment]
@mike-gangl I agree that it should't be open.  We should lock down the security group access, from only the HTTPD, and lock down the HTTPD only from the ALB, and so on, all the way back to the shared services HTTPD.  I'll create a new ticket to represent getting the MC locked down.
